Userland/DevTools/HackStudio/HackStudioWidget.cpp

```diff
@@ -1588,6 +1588,7 @@ void HackStudioWidget::update_tree_view()
 void HackStudioWidget::update_window_title()
 {
     window()->set_title(String::formatted("{} - {} - Hack Studio", m_current_editor_wrapper->filename_title(), m_project->name()));
+    window()->set_modified(any_document_is_dirty());
```

Comment:
This doesn't seem to be called reliably:
- Open a file and write into it, note that the window is not marked as modified
- Do that, switch to another document, switch back (note that it is marked as modified now), click the close button and select 'No'; note that the window is closed anyway.

Reply:
The `(*)` mark in the title is also not shown immediately (so we have an another bug). It seems that this function is called when opening a file. There should be a way to call it when modifying the file. Should I add some kind of callback which calls this?

Reply:
It think it works. How do I update the commit? I'm still new to this.